src/sentry/options/manager.py

```diff
@@ -39,17 +45,23 @@ class OptionsManager(object):
     Overall this is a very loose consistency model which is designed to give simple
     dynamic configuration with maximum uptime, where defaults are always taken from
     constants in the global configuration.
-
-    - Values must be strings.
-    - Empty values are identical to null values which are represented by ''.
     """
     cache = default_cache
-
     logger = logging.getLogger('sentry')
+    registry = {}
 
     # we generally want to always persist
     ttl = None
 
+    FLAG_DEFAULT = 0b000
```

Comment:
Oh, I see what you mean. This name is a little confusing because it's actually the default value for `flags` — not a bit mask like the `FLAG_` attributes below.